ui/src/app/pages/homepage/quick-tour-and-videos.tsx

```diff
@@ -118,12 +118,12 @@ export const QuickTourAndVideos = (props: Props) => {
             <div style={styles.quickTourAndVideosLabel}>Quick Tour and Videos</div>
             <div ref={quickTourResourcesDiv} style={{display: 'flex', position: 'relative'}}>
                 <FlexRow style={styles.quickTourCardsRow}>
-                    {quickTourResources.slice(quickTourResourceOffset, quickTourResourceOffset + limit()).map((thumbnail, i) => {
+                    {quickTourResources.slice(quickTourResourceOffset, quickTourResourceOffset + limit()).map((resource, i) => {
```

Comment:
`thumbnail` is a specific resource (which is in scope) so this name was not appropriate